frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigForm.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import InputLabel from '@material-ui/core/InputLabel';
+import { makeStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import uuid from "uuid";
```

Comment:
what is `uuid` used for?

Reply:
I was planning to use this when I make ids for the configurations

Reply:
I plan to use this when creating IDs for configurations.

Reply:
I don't think you should include this in this PR because it's not necessary for the code (in this PR) that you want to merge

Reply:
ya on a general note, it's good to import and include only things that are relevant for the specific PRs. :)